src/main/java/DeadlineCommand.java

```diff
@@ -0,0 +1,89 @@
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Command to create a deadline object.
+ */
+public class DeadlineCommand extends Command {
+
+    private String command;
+
+    /**
+     * Constructor method.
+     *
+     * @param command input command from user.
+     */
+    public DeadlineCommand(String command) {
+        this.command = command;
+    }
+
+    /**
+     * Execute method for deadline.
+     *
+     * @param taskList List of Tasks.
+     * @param ui Standard UI object.
+     * @param storage Standard storage object.
+     * @throws DukeMissingInputException If deadline command is missing description.
+     * @throws DukeWrongInputException If user input is not any of the commands available.
+     */
+    @Override
+    public String execute(TaskList taskList, Ui ui, Storage storage) throws DukeMissingInputException,
+            DukeWrongInputException {
+        String description = "";
+        String deadline = "";
+        boolean foundBy = false;
```

Comment:
Should name boolean variables to sound like booleans i.e hasBy of foundBy instead?

Reply:
I agree with you, this boolean variable should be made to sound like a boolean.